css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+
+* {
+  background-color: #1996da;
+  font-family: Lato Arial, Helvetica, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  color: white;
+  padding: 20px 40px;
```

Comment:
Only `width` and `margin` should be applied to the `container` class, you should make an additional class you can use to apply this styling.